Examples/StockQuoteConnector_final.html

```diff
@@ -31,26 +34,63 @@
     return num < 9 ? "0" + num.toString() : num.toString();
   }
 
+  function getTickersArrayFromConnectionData(connectionData) {
+    var tickersArray = connectionData.split(',');
+    
+    // Remove whitespace from each ticker string
+    return tickersArray.map(function(obj) {
+      return obj.trim(); 
+    });
+  }
+
   //
   // Connector definition
   // 
-
   var myConnector = tableau.makeConnector();
 
-  myConnector.getColumnHeaders = function() {
-    var fieldNames = ['Ticker', 'Day', 'Close'];
-    var fieldTypes = ['string', 'date', 'float'];
-    tableau.headersCallback(fieldNames, fieldTypes); // tell tableau about the fields and their types
+  myConnector.getSchema = function(schemaCallback) {
+    var tickersArray = getTickersArrayFromConnectionData(tableau.connectionData);
+    
+    var schema = [];
+    _.forEach(tickersArray, function(ticker) {
+      var col1 = { id: "Ticker", dataType: "string"};
+      var col2 = { id: "Day", dataType: "date"};
+      var col3 = { id: "Close", dataType: "float"};
+      var cols = [col1, col2, col3];
+      
+      var tableInfo = { 
+        id: ticker,
+        columns: cols
+      };
+      
+      schema.push(tableInfo);
+    });
+   
+    schemaCallback(schema); // tell tableau about the fields and their types
   };
    
-  myConnector.getTableData = function(lastRecordToken) {
+  myConnector.getData = function(tables, doneCallback) {
```

Comment:
Relatedly!  It might be nice to unify the done callbacks in a sort of node-ish way where it always takes an "error" as the first argument, followed by data (if necessary).  Would reduce the need for connector implementors to know how to handle errors (just pass in your error if you need to)...